lib/Sema/TypeCheckType.cpp

```diff
@@ -1032,6 +1032,50 @@ static std::string getDeclNameFromContext(DeclContext *dc,
   }
 }
 
+static Type isSE0068(TypeResolution resolution, TypeResolutionOptions options) {
+  auto dc = resolution.getDeclContext();
+  ASTContext &ctx = dc->getASTContext();
+  DeclContext *nominalDC = nullptr;
+  NominalTypeDecl *nominal = nullptr;
+  if ((nominalDC = dc->getInnermostTypeContext()) &&
+      (nominal = nominalDC->getSelfNominalTypeDecl())) {
```

Comment:
getSelfNominalTypeDecl() should never return null on a type context.

Reply:
You're beginning to nitpick the code before I got here.